### ISSUE TITLE
Running the Hello World example requires the --allow-env flag

### ISSUE THREAD

**Issue by cmd-johnson** [mandarineorg/mandarinets#199]
**[Details]**
When running the [Hello World example](https://www.mandarinets.org/docs/master/mandarine/hello-world), the application crashes before properly starting up, because it Mandarine (contradictory to the documentation) requires the `--allow-env` flag to work.

Using environment variables to configure Mandarine is fine, but I don't think having access to them should be mandatory.

Maybe it would be a good idea to try and read from the environment variables and when that fails because of missing permissions fall back to using a sane default value?

**[Stacktrace]**
```
error: Uncaught PermissionDenied: access to environment variables, run again with the --allow-env flag
    at Object.jsonOpSync (core.js:247:13)
    at Object.getEnv [as get] (deno:cli/rt/30_os.js:32:17)
    at new Log (log.ts:18:40)
    at Function.getLogger (log.ts:79:16)
    at renderEngine.ts:18:37
```

**[Scenario]**
1. Follow the [Hello World example](https://www.mandarinets.org/docs/master/mandarine/hello-world) from the Mandarine docs
    1. create a tsconfig.json file:
        ```
        {
          "compilerOptions": {
            "experimentalDecorators": true,
            "emitDecoratorMetadata": true
          }
        }
        ```
    2. Create a code file (myFile.ts)
        ```
        import { MandarineCore, Controller, GET } from "https://deno.land/x/mandarinets@v2.1.1/mod.ts";

        @Controller()
        export class MyController {

            @GET('/hello-world')
            public httpHandler() {
                return "Welcome to MandarineTS Framework!";
            }

        }

        new MandarineCore().MVC().run();
        ```
2. Run the application as described in the documentation
    `deno run -c tsconfig.json --allow-net --allow-read myFile.ts`

**[Expected behavior]**
The application starts up and returns `Welcome to MandarineTS Framework!` when making a GET request to `http://localhost:8080/hello-world`

**[OS]**
Linux (5.8.6-1 Manjaro)

**[Environment]**
 - Mandarine version: v2.1.1
 - Deno version: 1.4.2
 - Deno v8 version: 8.7.75
 - Deno typescript version: 4.0.3

**Comment by andreespirela** [comment]
Hi there, this is indeed a documentation issue @cmd-johnson . Mandarine internally makes use of `environmental variables` in order to work properly so it does need the `--allow-env` in order to work.

**Comment by andreespirela** [comment]
Closed in #206
